catroid/src/main/java/org/catrobat/catroid/content/bricks/VibrationBrick.java

```diff
@@ -71,54 +57,10 @@ public int getViewResource() {
 		return R.layout.brick_vibration;
 	}
 
-	@Override
-	public View getView(Context context) {
-		super.getView(context);
-		TextView editVibrate = (TextView) view.findViewById(R.id.brick_vibration_edit_text);
-		TextView secondTextVibrate = (TextView) view.findViewById(R.id.brick_vibration_second_label);
```

Comment:
Please also use your new `setSecondsLabel` here.

Reply:
oh, well, interestingly this one did not come up on the `BrickValueParameterTest` :thinking:. someone should have a look at this test again anyway -> @84n4n4 maybe?